### PR TITLE
Create dt_list.py

### PR DESCRIPTION
**Pull request by sav-norem**
as part of the effort to get tabbed examples in client languages on the redis.io website, this file contains the examples for the list datatype in python with the appropriate short codes to be pulled in

### CODE REVIEW

**Comment by sav-norem** [comment]
@uglide and lists!

File: doctests/dt_list.py [hunk 1]
```diff
@@ -0,0 +1,324 @@
+# EXAMPLE: list_tutorial
+# HIDE_START
+import redis
+
+r = redis.Redis(decode_responses=True)
+r.delete("bikes:repairs")
```
Comment (uglide) [inline doctests/dt_list.py:6]: Please wrap those two lines in the REMOVE_START/END block to hide this cleanup code from the users.

File: doctests/dt_list.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+r.delete("bikes:finished")
+# HIDE_END
+
+# STEP_START queue
```
Comment (uglide) [inline doctests/dt_list.py:10]: See my comment here https://github.com/redis/redis-doc/pull/2488/files#r1269382572

**Review by uglide — CHANGES_REQUESTED** [review]
Great work!
We can make this example more concise by explaining List basics around one pattern Producer/Consumer which is extensively covered in the current version of the doc here https://redis.io/docs/data-types/lists/

**Comment by sav-norem** [comment]
@uglide I believe those changes would be better addressed with an issue for the docs team to rework the structure of that content

**Comment by sav-norem** [comment]
@chayim if you could take a look!

**Pull request merged**